pkgs/servers/apache-directory-server/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, adoptopenjdk-jre-openj9-bin-11, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "apache-directory-server";
+  version = "2.0.0.AM26";
+
+  src = builtins.fetchTarball {
```

Comment:
```suggestion
  src = fetchurl {
```

Reply:
Thanks for the suggestion.

What is the benefit of `fetchurl` over `fetchTarball`?

If my understanding is correct, `fetchurl` will fetch the tarball and place it in `$src`, so then I have to unpack it manually. It also needs to be imported from pkgs, but `fetchTarball` is built in.

Reply:
`builtins.fetchTarball` is not allowed in restricted mode, see [here](https://discourse.nixos.org/t/why-is-fetchtarball-not-mentioned-in-chapter-11-fetchers-of-the-nixpkgs-manual/15319/2).

While the direct equivalent is `pkgs.fetchzip`, I think using `pkgs.fetchurl` is also possible, since the tarball will still be automatically unpacked in `unpackPhase`.

Reply:
Thank you for the helpful explanation. I've tried `fetchurl` and it did not seem to unpack automatically. So I went with `fetchzip` instead.